packages/odyssey/src/scss/base/_typography-text.scss

```diff
@@ -61,6 +61,12 @@ p {
   }
 }
 
+pre {
+  font-family: $mono-font-family;
+  white-space: pre-wrap;
+  tab-size: 2;
```

Comment:
Our tab size is 2, so... 😎

Reply:
I'm not sure how much Okta _has_ standardized on a tab format (looks like some of our SDKs vary between 2 and 4 spaces), but I very much favor having a standard.